### ISSUE TITLE
Best way to install/update Nightly

### ISSUE THREAD

**Issue by Infallibleman** [SchizoDuckie/DuckieTV#1352]
Hello, 

What is the best way to install / Update Nightly installs?  Just run the package installer and it will automatically update/overwrite  the existing install or a recommended method to completely remove the existing install and then do a fresh one?

I want to ensure we don't have any orphaned/remnant files as part of this exercise that accumulate over time.

Thank you,
H

**Comment by garfield69** [comment]
TLDR; just run the installer. (On a macOS, first drag the DuckieTV app from the Applications folder to the Bin).

As you may know, Dtv is written as a Chromium Browser Extension, which is just Javascript written using the AngularJS framework, with supporting HTML templates and CSS.

The Standalone version of Dtv is the exact same JS encapsulated by the NWJS run time environment, which contains a headless Chromium browser so Dtv can run as a program independently.
 
So the Dtv Nightly standalone installer simply overwrites the folder where the NWJS program keeps the DuckieTV JS, HTML and CSS and NWJS modules.
The Dtv Data files are not touched, as these are at a different location controlled by the Chromium engine.
